plugins/cpu/fu-plugin-cpu.c

```diff
@@ -46,6 +47,9 @@ fu_plugin_coldplug (FuPlugin *plugin, GError **error)
 			data->has_cet = TRUE;
 		if (fu_cpu_device_has_flag (dev, FU_CPU_DEVICE_FLAG_TME))
 			data->has_tme = TRUE;
+		if (fu_cpu_device_has_flag (dev, FU_CPU_DEVICE_FLAG_SMEP) &&
+		    fu_cpu_device_has_flag (dev, FU_CPU_DEVICE_FLAG_SMAP))
+			data->has_smap = TRUE;
```

Comment:
At least the way the wikipedia page reads, SMAP is the extension to SMEP.  I think you just want to look for SMAP.

Reply:
So maybe just ignore SMEP completely?

Reply:
Assuming wikipedia is right, yeah I think so.

Reply:
Gotcha, fixed.